### DISCUSSION TITLE
MountainDuck : sftp mount shows no files on macOS Ventura 13.4 (22F66) -> OpenSSH_9.1p1, OpenSSL 1.1.1s  1 Nov 2022

### DISCUSSION THREAD

**Discussion by greycubesgav** [iterate-ch/cyberduck#14821] [Q&A]
I am experiencing a regular issue with a new Trial of MountainDuck.

When connecting to an sftp connection (vOpenSSH_9.1p1 on local lan) the finder window is empty showing no files after connecting. Connecting through terminal (in mountain duck) shows files.

Process to Repeat:
* Open Mountain Duck
* Connect to local sftp connection
* Finder window opens showing no files
* Disconnecting from the share and reconnecting does not resolve the issue.
* Quitting MountainDuck and restarting sometimes resolves the issue.

The issue is intermittent, where by it will consistently show no files, then on another load it will show files.

I also have an issue where when it does show the list of directories, when opening a directory the directory listing will also be empty.

I've attached some gifs to try and show the issue in action.

Mounting the share, showing 0 files:
![2023-06-18 17 16 39](https://github.com/iterate-ch/cyberduck/assets/3426859/43e5e0ce-b584-4e7e-b1bf-bcf0c22b0dd5)

Listing the same share though sftp on the terminal:
![2023-06-18 17 17 25](https://github.com/iterate-ch/cyberduck/assets/3426859/33abf88a-be93-41c0-8573-a01728198733)

Browsing to the local mounted directory under terminal shows the same issue, no files, or no fils in a sub-directory.

Is this a known issue with sftp?
Is there a way to 'force' a refresh of the mount?

Thanks

**Comment by greycubesgav** [comment]
It seems to be related to the "Connect Mode", settings this explicitly to "Online" allow the files to be shown every time.

<img width="390" alt="Screenshot 2023-06-18 at 19 15 56" src="https://github.com/iterate-ch/cyberduck/assets/3426859/2ad651d7-26af-4dd9-918d-2d0fb35d9328">

It looks like the issue is just with "Smart Syncronization" not showing folders / files?

**Comment by dkocher** [comment]
We would need to have access to a _Debug_ log to have a closer look. Please choose _Preferences → Connection → Log → Show_ and attach the selected file {{mountainduck.log}}.
